lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -5429,20 +5451,25 @@ class ConvertTorchToTosa : public ConvertTorchToTosaBase<ConvertTorchToTosa> {
     target.addLegalOp<ConstantDeviceOp>();
     target.addLegalOp<PrimListConstructOp>();
     target.addLegalOp<PrimTupleConstructOp>();
-    target.addIllegalDialect<Torch::TorchDialect>();
+    if (!bypassIllegalOpsCheck)
```

Comment:
Could you please add few comments here clarifying what `illegal` dialects are in this context? This will help if someone looks at this code later.

Reply:
I don't want to modify existing pass to add this info but essentially the idea is after `torch-to-tosa` there shouldn't be any `torch` ops left.